libs/dyn/jsonloader/json.go

```diff
@@ -0,0 +1,21 @@
+package jsonloader
+
+import (
+	"encoding/json"
+
+	"github.com/databricks/cli/libs/dyn"
+)
+
+func LoadJSON(data []byte) (dyn.Value, error) {
+	var root map[string]interface{}
+	err := json.Unmarshal(data, &root)
+	if err != nil {
+		return dyn.InvalidValue, err
+	}
+
+	loc := dyn.Location{
+		Line:   1,
+		Column: 1,
+	}
+	return newLoader().load(&root, loc)
```

Comment:
What would it take to have real location information here?

Reply:
@pietern this is what I am working on still, so TBD :)

Reply:
Added here https://github.com/databricks/cli/pull/1771/commits/0aca3741442667b3b812dcfc52a0e59900e3dbe2